### PR TITLE
tests: add test for cprint package

### PR DESCRIPTION
**Pull request by hbagdi**

### CODE REVIEW

File: cprint/color_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	go func() {
+		var buf bytes.Buffer
+		defer wg.Done()
+		io.Copy(&buf, reader)
+		out <- buf.String()
+	}()
+	defer wg.Wait()
+	f()
+	writer.Close()
+	return <-out
```
Comment (mflendrich) [inline cprint/color_test.go:33]: Won't this work:

```go
var out bytes.Buffer
color.Output = out
f()
return out.String()
```

?

Comment (hbagdi) [reply cprint/color_test.go]: Works!
It is amazing how my tunnel vision led me to this ugly monster.

File: cprint/color_test.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/fatih/color"
+	"github.com/stretchr/testify/assert"
+)
+
+func captureOutput(f func()) string {
```
Comment (mflendrich) [inline cprint/color_test.go:12]: Sorry about bugging you again - forgot about this. We should probably revert the prior setting of `color.Output` and `defer` setting it back.

**Review by mflendrich — APPROVED** [review]
LGTM but see the comment I didn't think of during the first round of review.

File: cprint/color_test.go (hunk 2 above)
Comment (hbagdi) [reply cprint/color_test.go:15]: Updated!

**Pull request merged**